src/ui/czi-ordered-list-counters.scss

```diff
@@ -0,0 +1,12 @@
+// prettier-ignore
+
+// This generate many rules for resetiing the counter list with specific
+// start number. Ideally we could just use one line of CSS like
+// `.ProseMirror ol[start] { counter-reset: czi-counter attr(start); }
+// but unfortunately no browser ever support it so we could only use SASS to
+// do it.
+@for $i from 1 to 500 {
```

Comment:
please elaborate on the purpose of this - what is the expected styling?

Reply:
The output is `src/ui/czi-ordered-list-counters.css`.
I'm not sure I should just type the generated CSS here cause the SASS should explain itself.

Reply:
What I was asking is what should the result be in the styling - screenshots etc would be helpful.

Reply:
And honestly if we're doing this in a single file I would greatly prefer you just write it yourself in css.